instrumentation/active_model_serializers/lib/opentelemetry/instrumentation/active_model_serializers/instrumentation.rb

```diff
@@ -35,7 +35,7 @@ def require_dependencies
         end
 
         def register_event_handler
-          ActiveSupport::Notifications.subscribe(event_name) do |_name, start, finish, _id, payload|
+          ::ActiveSupport::Notifications.subscribe(event_name) do |_name, start, finish, _id, payload|
```

Comment:
Is there a way to write encode this in an automated test case that will reveal this error and avoid regressions?

Reply:
Well, I thought about it. But it is reproduced only when both `opentelemetry-instrumentation-active_support` and `opentelemetry-instrumentation-active_model_serializers` are included to the app, and these gems have isolated test specs. So it's a bit hard to test the conflict.

Reply:
I wonder if this is something we may be able to detect in the `all` test suite? cc: @open-telemetry/ruby-approvers

Reply:
Done! Reproduced with a quite simple test case. Thanks for the suggestion 🙂

Reply:
Sweet! Did the tests fail when you removed the top level qualifier `::`?

Reply:
Yep, that's what I started from.

Reply:
```
Finished in 0.006173s, 161.9958 runs/s, 0.0000 assertions/s.

  1) Error:
OpenTelemetry::Instrumentation::All::ActiveModelSerializers instrumentation#test_0001_does not conflict with ActiveSupport instrumentation:
NameError: uninitialized constant OpenTelemetry::Instrumentation::ActiveSupport::Notifications
```